scripts/benchmarks-ci.sh

```diff
@@ -64,6 +64,10 @@ elif [[ $runtimeName == "bridge-hub-rococo" ]]; then
                 cumulus_pallet_xcmp_queue
                 pallet_xcm_benchmarks::generic
                 pallet_xcm_benchmarks::fungible
+                # snowbridge_basic_channel::inbound
+                snowbridge_basic_channel::outbound
+                # snowbridge_dispatch
```

Comment:
`snowbridge_basic_channel::inbound` and `snowbridge_dispatch` require benchmarking.